apps/api-gateway/src/auth/guards/jwt-refresh-auth.guard.ts

```diff
@@ -0,0 +1,9 @@
+import { Injectable } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+
+@Injectable()
+export class JwtRefreshGuard extends AuthGuard('jwt-refresh') {
```

Comment:
이렇게 가야 파일명이랑 동일할 것 같아요~
```suggestion
export class JwtRefreshAuthGuard extends AuthGuard('jwt-refresh') {
```

Reply:
[4dbb902](https://github.com/GrebbMe/grabbme-be/pull/53/commits/4dbb902350c8073ed7fbe7f16bb0c71fae45cf53)
위 커밋에 반영하였습니다!